src/test/python/test_srt_reader.py

```diff
@@ -205,7 +206,31 @@ def test_long_hours(self):
       363601,
       doc.get_body().first_child().first_child().get_end()
     )
-    
+
+  def test_starts_with_span_single_line(self):
+    f = io.StringIO(r"""1
+101:00:00,000 --> 101:00:01,000
+Hello
+""")
+    doc = to_model(f)
+
+    self.assertIsInstance(
+      doc.get_body().first_child().first_child().first_child(),
```

Comment:
I suggest expanding the test to make sure there is only line that contains _Hello_.

Reply:
Thank you, updated test.